app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -291,7 +291,9 @@ a.btn-info span{
   padding-bottom: 4px;
 }
 
-span.status-error, span.status-ok, span.status-warn, span.status-question {
-  padding-left: 3px;
+#properties_table {
+  span.status-error, span.status-ok, span.status-warn, span.status-question {
```

Comment:
This also adds the padding to the text, which has the same class. Why not just `.host-status` which will select just the icon?